src/WebJobs.Extensions.DurableTask/EndToEndTraceHelper.cs

```diff
@@ -11,12 +12,26 @@ internal class EndToEndTraceHelper
         private static string appName;
         private static string slotName;
 
-        // TODO: Replace TraceWriter with structured event client
-        private readonly TraceWriter appTraceWriter;
+        private readonly ILogger logger;
 
-        public EndToEndTraceHelper(TraceWriter appTraceWriter)
+        public EndToEndTraceHelper(JobHostConfiguration config)
         {
-            this.appTraceWriter = appTraceWriter;
```

Comment:
If you want these logs going to the host log files, you'll need to double-log with the TraceWriter as well.

Reply:
Is this a POT thing or is this the long term intention?

Reply:
For v2 -- point-in-time. I want to move everything to ILogger soon.
For v1 -- not sure. If it's a simple port backwards, then we may change it there as well.

Reply:
v2 is what matters most